markdownfmt/testfiles/lists.output.md

```diff
@@ -72,7 +72,6 @@ Nested:
    2. two
    3. three
 2. untight
-
```

Comment:
this is probably fine, I will need to google tight/untight markdown lists again later

Reply:
I will look into `The logic is related to tight vs. untight markdown lists.` as well (:

Reply:
tight/loose is the correct wording

Reply:
Okey I dived a bit into this details.

I would take GFM as an opinionated spec we can  rely on (hope that makes sense) and [there is](https://github.github.com/gfm/#tight):

> A list is loose if any of its constituent list items are separated by blank lines, or if any of its constituent list items directly contain two block-level elements with a blank line between them. Otherwise, a list is tight. (The difference in HTML output is that paragraphs in a loose list are wrapped in <p> tags, while paragraphs in a tight list are not.)

I would say there is no strict specific on this particular, but I would propose to leave this as it. In fact in theory if there is a "gap" it will mean the parent list should be loose as well, right? 🤔 

Let's fix it later if needed. The reason is that the implementation would need to be extra complex if we want to customize it here (: